sdk/media/azure-media-lva-edge/README.md

```diff
@@ -0,0 +1,127 @@
+
+## Examples
+
+### Creating a graph topology
+To create a graph topology you need to define parameters, sources, and sinks.
+```
+#Parameters
+user_name_param = MediaGraphParameterDeclaration(name="rtspUserName",type="String",default="dummyusername")
+password_param = MediaGraphParameterDeclaration(name="rtspPassword",type="String",default="dummypassword")
+url_param = MediaGraphParameterDeclaration(name="rtspUrl",type="String",default="rtsp://www.sample.com")
+
+#Source and Sink
+source = MediaGraphRtspSource(name="rtspSource", endpoint=MediaGraphUnsecuredEndpoint(url="${rtspUrl}",credentials=MediaGraphUsernamePasswordCredentials(username="${rtspUserName}",password="${rtspPassword}")))
+node = MediaGraphNodeInput(node_name="rtspSource")
+sink = MediaGraphAssetSink(name="assetsink", inputs=[node],asset_name_pattern='sampleAsset-${System.GraphTopologyName}-${System.GraphInstanceName}', segment_length="PT0H0M30S",local_media_cache_maximum_size_mi_b=2048,local_media_cache_path="/var/lib/azuremediaservices/tmp/")
+
+graph_properties = MediaGraphTopologyProperties(parameters=[user_name_param, password_param, url_param], sources=[source], sinks=[sink], description="Continuous video recording to an Azure Media Services Asset")
+
+graph_topology = MediaGraphTopology(name=graph_topology_name,properties=graph_properties)
+
+```
+
+### Creating a graph instance 
+To create a graph instance, you need to have an existing graph topology.
+```
+url_param = MediaGraphParameterDefinition(name="rtspUrl", value=graph_url)
+graph_instance_properties = MediaGraphInstanceProperties(description="Sample graph description", topology_name=graph_topology_name, parameters=[url_param])
+
+graph_instance = MediaGraphInstance(name=graph_instance_name, properties=graph_instance_properties)
+
+```
+
+### Invoking a graph method request
+To invoke a graph method on your device you need to first define the request using the lva sdk. Then send that method request using the iot sdk's `CloudToDeviceMethod`
+```
+set_method_request = MediaGraphTopologySetRequest(graph=graph_topology)
+direct_method = CloudToDeviceMethod(method_name=set_method_request.method_name, payload=set_method_request.serialize())
+registry_manager = IoTHubRegistryManager(connection_string)
+
+registry_manager.invoke_device_module_method(device_id, module_d, direct_method)
+```
+
+For more samples please visit [Samples][samples].
```

Comment:
Can we make it a bit more explicit, like, To try different media graph topologies with the SDK, visit....